src/Http/Controllers/Auth/SsoController.php

```diff
@@ -145,6 +151,20 @@ private function findOrCreateUser(SocialiteUser $eve_user): User
             return $existing;
         }
 
+        // Detect if registration is disabled
+        // Abort user creation.
+        if (setting('registration', true) === 'no') {
+
+            // Log the account creation attempt
+            event('security.log', [
+                'Attempt of account creation for ' . $eve_user->name, ' blocked, as registration is turned off.',
```

Comment:
Reword to:

```
Account creation for {user} failed. Registration has been disabled.
```

This should probably also be a translation candidate.

Reply:
maybe in a later PR as it would require to exchange the variable dynamically